frontend/src/js/snack-message/SnackMessage.tsx

```diff
@@ -7,13 +7,20 @@ import { useClickOutside } from "../common/helpers/useClickOutside";
 import FaIcon from "../icon/FaIcon";
 
 import { setMessage } from "./actions";
+import { SnackMessageStateT, SnackMessageTypeT } from "./reducer";
 
-const Root = styled("div")`
+const colorLookupTable: { [key in SnackMessageTypeT]: string } = {
+  [SnackMessageTypeT.ERROR]: "rgba(0, 0, 0, 0.75)",
+  [SnackMessageTypeT.SUCCESS]: "rgba(12, 100, 39, 0.75)", // #0C6427
```

Comment:
Have you tested what it looks like? I think with the green, it's actually too transparent, how about using `0.9`?

Reply:
I liked it that way. Took a look at 0.9, you are right, that looks better